themes/upbound/assets/scss/bootstrap/_variables.scss

```diff
@@ -630,11 +630,11 @@ $line-height-base:            1.5 !default;
 $line-height-sm:              1.25 !default;
 $line-height-lg:              2 !default;
 
-$h1-font-size:                $font-size-base * 2.5 !default;
-$h2-font-size:                $font-size-base * 2 !default;
-$h3-font-size:                $font-size-base * 1.75 !default;
-$h4-font-size:                $font-size-base * 1.5 !default;
-$h5-font-size:                $font-size-base * 1.25 !default;
+$h1-font-size:                $font-size-base * 2 !default;
+$h2-font-size:                $font-size-base * 1.5 !default;
+$h3-font-size:                $font-size-base * 1.25 !default;
+$h4-font-size:                $font-size-base * 1 !default;
+$h5-font-size:                $font-size-base * 1 !default;
```

Comment:
This makes these headings not stand out. [Example](https://docs-git-fork-teich-designv3-upboundio.vercel.app/concepts/managed-control-planes/#configure-crossplane-providers-on-your-mcp)

I'm okay with it, just making sure we're aware.